content/en/service_management/service_level_objectives/guide/slo_types_comparison.md

```diff
@@ -0,0 +1,53 @@
+---
+title: SLO Type Comparison
+kind: Guide
+further_reading:
+- link: "/service_management/service_level_objectives/"
+  tag: "Documentation"
+  text: "Overview of Service Level Objectives"
+- link: "/service_management/service_level_objectives/metric/"
+  tag: "Documentation"
+  text: "Metric-based SLOs"
+- link: "/service_management/service_level_objectives/monitor/"
+  tag: "Documentation"
+  text: "Monitor-based SLOs"
+- link: "/service_management/service_level_objectives/time_slice/"
+  tag: "Documentation"
+  text: "Time Slice SLOs"
+---
+
+## Overview
+
+When creating SLOs, you can choose from the following types:
+- **Metric-based SLOs**: can be used for count-based data streams, the SLI is based on the sum of good events divided by the sum of total events.
+- **Monitor-based SLOs**: can be be used for time-based data sets, the SLI is based on the amount of time your system exhibits good behavior divided by the total time. Monitor-based SLOs must be based on a new or existing Datadog monitor, any adjustments must be made to the underlying monitor (cannot be done through SLO creation).
```

Comment:
It's not really true that the data set needs to be "time based". That describes the calculation but not the data. They can use can data at all. It would be more correct to say that the SLI is based on the monitor's uptime (times when the monitor is not in the "Alert" state).